Code/Robot/line_follow.json

```
@@ -0,0 +1,13 @@
+{
+  "g": [
+    {
+      "type": "calibrate",
+      "duration": 6000
+    },
+    {
+      "type": "linefollow",
+      "stopDistance": 10,
+      "speed": 200,
+    }
+  ]
+}
```

```suggestion
{
  "g": [
    {
      "type": 4,
      "duration": 6000
    },
    {
      "type": 1,
      "stopDistance": 10,
      "speed": 200
    }
  ]
}
```

No trailing comma in json, this file is invalid json lmao